plutus-core/executables/src/PlutusCore/Executable/Parsers.hs

```diff
@@ -100,7 +100,7 @@ printmode :: Parser PrintMode
 printmode = option auto
   (  long "print-mode"
   <> metavar "MODE"
-  <> value Simple
+  <> value Classic
   <> showDefault
   <> help
     ("Print mode for textual output (ignored elsewhere): Classic -> plcPrettyClassic, "
```

Comment:
I'm still confused about the names and I'm too lazy to look at the code.  I think that (maybe) `evaluate` always prints the AST that it gets back from the CEK machine, and that has de Bruijn indices (although it may or may not showthem to you depending on the print mode).  I think the confusing thing is that we also have `convert` which lets you change the name type, but you can only do that if you output flat, and then maybe you can convert that to textual form.  Do we have any way of converting the output of a program to a textual form that you can sucessfully parse again?

Also, did we ever talk about distinguishing indices from uniques in textual output?  Maybe part of the confusion is that they look exactly the same.

Reply:
> Also, did we ever talk about distinguishing indices from uniques in textual output? Maybe part of the confusion is that they look exactly the same.

DeBruijn indexes [use `!` as a separator,](https://github.com/IntersectMBO/plutus/blob/66b240239e1afd8ef8a72791c3233300745db0b8/plutus-core/plutus-core/src/PlutusCore/DeBruijn/Internal.hs#L174) while [uniques use `-`.](https://github.com/IntersectMBO/plutus/blob/66b240239e1afd8ef8a72791c3233300745db0b8/plutus-core/plutus-core/src/PlutusCore/Name/Unique.hs#L109-L110)